import-dashboards.py

```diff
@@ -529,9 +530,84 @@ def set_home_dashboard(api_key):
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
 
+def dbaas_dashboard(api_key):
+    if DBAAS is None or DBAAS.lower() in ('0','false','f'):
+        print ' * DBaaS is disabled'
+        r = requests.get('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+        if r.status_code == httplib.OK:
+            print '   * Removing DBaaS dashboard'
+            r = requests.delete('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+            if r.status_code != httplib.OK:
+                print r.status_code, r.content
+        return
+
+    print ' * DBaaS is enabled'
+    r = requests.get('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+    if r.status_code != httplib.OK:
+        print '   * Adding DBaaS dashboard'
+        with open('/usr/share/percona-dashboards/pmm-app/dist/dashboards/pmm-dbaas.json', 'r') as dashboard_file:
+            dashboard = json.loads(dashboard_file.read())
+        data = json.dumps({'dashboard': dashboard})
+        r = requests.post('%s/api/dashboards/import' % (HOST,), data=data, headers=grafana_headers(api_key))
+        if r.status_code != httplib.OK:
+            print r.status_code, r.content
+    else:
+        print '   * DBaaS dashboard has already been added'
+
+
+def check_active_process(processname):
+    ps = os.popen("ps -Af").read()
+    proccount = ps.count(processname)
+
+    if proccount > 0:
+        print' * %s service is running' % (processname,)
+        return True
+
+
+def args_parser():
+   parser = argparse.ArgumentParser()
+   parser.add_argument("grafana_db_dir", type=str, nargs='?', default="/srv/grafana", help="Set Grafana DB folder")
+   parser.add_argument("-d", "--dbaas", required=False, help="Perform only operation of add/remove dbass dashboard", action="store_true")
```

Comment:
I don't suggest we use the short version of `-d` - it's just not explicit enough. Better to always use the long version `--dbaas`- nobody has to guess what it is.

Reply:
The short flag is removed.